include/circt/Conversion/Passes.td

```diff
@@ -813,6 +813,11 @@ def ConvertCombToAIG: Pass<"convert-comb-to-aig",  "hw::HWModuleOp"> {
     "circt::comb::CombDialect",
     "circt::aig::AIGDialect",
   ];
+
+  let options = [
+    ListOption<"additionalLegalOps", "additional-legal-ops", "std::string",
+               "Sepify additional legal ops for testing">,
```

Comment:
Also- is anything using this feature in this PR? Normally, I think it's a little unusual to add testing flags to passes that aren't testing passes, but depending on how it's used this might make sense.

Reply:
Thanks, I missed `git add` for the test. Added `test/Conversion/CombToAIG/comb-to-aig-arith.mlir`.